apps/coscrad-frontend-e2e/src/e2e/ux-components/multilingual-text-presenter.e2e.cy.ts

```diff
@@ -0,0 +1,64 @@
+import { AggregateType, LanguageCode, MIMEType } from '@coscrad/api-interfaces';
+import { buildDummyAggregateCompositeIdentifier } from '../../support/utilities';
+
+const mediaItemName = 'media item for the video';
+
+const mediaItemCompositeIdentifier = buildDummyAggregateCompositeIdentifier(
+    AggregateType.mediaItem,
+    2
+);
+
+const validUrl =
+    'https://coscrad.org/wp-content/uploads/2023/05/Rexy-and-The-Egg-_3D-Dinosaur-Animation_-_-3D-Animation-_-Maya-3D.mp4';
+
+const videoNameInEnglish = `Video 1`;
+
+const videoAggregateCompositeIdentifier = buildDummyAggregateCompositeIdentifier(
+    AggregateType.video,
+    1
+);
+
+const { id: videoId } = videoAggregateCompositeIdentifier;
+
+const videoDetailRoute = `/Resources/Videos/${videoId}`;
+
+describe('Multilingual Text Presenter', () => {
+    before(() => {
+        cy.clearDatabase();
+
+        cy.seedTestUuids(10);
+
+        cy.executeCommandStreamByName('users:create-admin');
+
+        cy.seedDataWithCommand(`CREATE_MEDIA_ITEM`, {
+            aggregateCompositeIdentifier: mediaItemCompositeIdentifier,
+            titleEnglish: mediaItemName,
+            mimeType: MIMEType.mp4,
+            url: validUrl,
+        });
+
+        cy.seedDataWithCommand(`PUBLISH_RESOURCE`, {
+            aggregateCompositeIdentifier: mediaItemCompositeIdentifier,
+        });
+
+        cy.seedDataWithCommand(`CREATE_VIDEO`, {
+            aggregateCompositeIdentifier: videoAggregateCompositeIdentifier,
+            name: videoNameInEnglish,
+            languageCodeForName: LanguageCode.English,
+            mediaItemId: mediaItemCompositeIdentifier.id,
+            lengthMilliseconds: 720000,
+        });
+
+        cy.seedDataWithCommand(`PUBLISH_RESOURCE`, {
+            aggregateCompositeIdentifier: videoAggregateCompositeIdentifier,
+        });
```

Comment:
To be thorough, we'll want to have a test for when
- the video has an original name in English
    - with a `defaultLanguageCode` translation
    - without any translation
- the video has an original name in the `defaultLanguageCode` (be careful that the test doesn't depend on this config!
    - with an English translation
    - without an English translation
 And assert that the text for each language is displayed property.

Reply:
Ok cool, I forgot how to test for other variations in the multilingual text system.  Hard to keep my head around...